gotocompany/compass/v1beta1/service.proto

```diff
@@ -1055,6 +1069,24 @@ message GetAllAssetsRequest {
   bool with_total = 10 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
     description: "if set include total field in response"
   }];
+  string duration_for_upper_threshold = 11 [
```

Comment:
why do we need the changes in the following line?

Reply:
Currently, my idea is Delete Assets will use **Filter** type in Compass (same as GetAllAssets), with mandatory filled for duration_for_upper_threshold field.

So, I think it is okay to make GetAllAssets also can filter by time (refreshed_at, updated_at, or created_at), since use same Filter type in Compass.

Reply:
let's skip this changes for now? if needed can create separate PR

Reply:
if we plan to use duration, let's use `google.protobuf.Duration` instead of `string`

Reply:
okayy